packages/codemods/src/command.ts

```diff
@@ -35,25 +38,67 @@ export const yargsParse = (passedArgs: string[]) => {
         'switches the filters from being inclusive, to exclusive.' +
         'If no filters provided, will exclude all mods by default',
     })
+    .option('config', {
+      alias: 'c',
+      type: 'boolean',
+      default: false,
+      description: 'switches modrunner to get args from the config file modConfig.json. False by default.',
+    })
     .parse(passedArgs);
 };
-
+/* Class responsible for parsing the npx command that runs codemods on a repo.
+   Can either take info from the command line or from a config file. */
 export class CommandParser {
   public parseArgs(passedIn: string[]): CommandParserResult {
     const parsed = yargsParse(passedIn);
     if (parsed.help) {
       return { shouldExit: true, modsFilter: () => true };
     }
+    let configObj: ModRunnerConfigType = { stringFilters: [], regexFilters: [], includeMods: false };
+    if (parsed.config) {
+      /* Attempt to locate the modConfig file in the user's repo. */
+      const configResult = runConfig();
+      if (configResult.ok) {
+        configObj = configResult.value;
+      } else {
+        console.log(configResult.value);
+        return { shouldExit: true, modsFilter: () => true };
+      }
+    }
+
     const filts = {
-      stringFilter: Maybe(parsed.modNames),
-      regexFilter: Maybe(parsed.modPatterns),
+      stringFilter: parsed.config ? Maybe(configObj.stringFilters) : Maybe(parsed.modNames),
+      regexFilter: parsed.config ? Maybe(configObj.regexFilters) : Maybe(parsed.modPatterns),
     };
 
-    const filter = parsed.excludeMods ? getModExcludeFilter(filts) : getModFilter(filts);
+    const shouldExclude: boolean = parsed.config ? !configObj.includeMods : parsed.excludeMods;
+    const filter = shouldExclude ? getModExcludeFilter(filts) : getModFilter(filts);
 
     return {
       shouldExit: false,
       modsFilter: filter,
     };
   }
 }
+
+function runConfig(): Result<ModRunnerConfigType, NoOp> {
+  const foundJsonFile = getModRunnerConfig();
```

Comment:
IMO the whole part of this runConfig could be returned from getModRunnerConfig()